src/applications/disability-benefits/686c-674/config/chapters/add-a-child/child-information/helpers.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export function childInfo({ formData }) {
```

Comment:
We should think about refactoring all of these helpers..I write a lot of the same things for workflows I'm doing 😅